### PR TITLE
Add hebrew-sentiment

### PR DESCRIPTION
**Pull request by elronbandel**
hebrew-sentiment dataset is ready! (including tests, tags etc)

### CODE REVIEW

File: datasets/hebrew-sentiment/hebrew-sentiment.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
+"""HebrewSentiment: A Modern Hebrew Sentiment Analysis Dataset."""
```
Comment (lhoestq) [inline datasets/hebrew-sentiment/hebrew-sentiment.py:1]: could you rename the paths using `hebrew_sentiment` ?
we're using snake case convention

**Review by lhoestq — COMMENTED** [review]
Nice ! Looking very good
Thank you for adding this dataset :)

I just left one comment about the naming of the folder and the python file of the dataset

File: datasets/hebrew-sentiment/hebrew-sentiment.py (hunk 1 above)
Comment (elronbandel) [reply datasets/hebrew-sentiment/hebrew-sentiment.py:1]: Done.

**Comment by yjernite** [comment]
@elronbandel it looks like something went wrong with the renaming, as the old files are still in the PR. Can you `git rm datasets/hebrew-sentiment` ?

**Review by lhoestq — APPROVED** [review]
thanks ! LGTM :)

**Comment by lhoestq** [comment]
merging since the CI is fixed on master

**Comment by shaked571** [comment]
This is the old version of the data.
Here is the fixed version.
https://github.com/OnlpLab/Hebrew-Sentiment-Data

**Comment by lhoestq** [comment]
Cool ! Sure feel free to open a PR if you have some time :) and feel free to ping me for review or if you have questions

**Pull request merged**